spec/features/admin/merchants/index_spec.rb

```diff
@@ -27,4 +27,32 @@
 
     expect(current_path).to eq("/admin/merchants/#{@merchant_1.id}")
   end
-end
\ No newline at end of file
+
+  it 'has an interface to enable or disable merchants' do
+    @merchant_2.destroy
+    @merchant_3.destroy
+    @merchant_4.destroy
+    @merchant_5.destroy
+
+    visit admin_merchants_path
+
+    within('#disabled_merchants') do
+      expect(page).not_to have_button("Disable")
```

Comment:
we could nest a within `"merchant_#{merchant.id}"` block inside `within #disabled_merchants` to test if a specific merchant is being changed